app/server/utils/blockchain_tasks.py

```diff
@@ -37,9 +35,7 @@ def _execute_synchronous_celery(
             async_result.forget()
         return response
 
-    def _synchronous_call(
-            self, contract_address, contract_type, func, args=None, signing_address=None, queue='high-priority'
-    ):
+    def _synchronous_call(self, contract_address, contract_type, func, args=None, signing_address=None, queue='high-priority'):
```

Comment:
we definitely need a python style enforcer lol. Because my IDE yells at me for the new version!